console/attribute_profile.go

```diff
@@ -25,7 +25,7 @@ import (
 
 func init() {
 	c := &CmdGetAttributes{
-		name:      "attributes",
+		name:      "attribute_profile",
```

Comment:
Thinking it should be nicer if we could rename to attributes_profile (like the subsystem). This should apply for all the commands. Need to review together since I am not sure.